src/TMxpProcessor.h

```diff
@@ -35,23 +36,11 @@ enum TMxpProcessingResult { HANDLER_FALL_THROUGH, HANDLER_NEXT_CHAR, HANDLER_COM
 // handles the MXP protocol
 class TMxpProcessor
 {
-    // State of MXP system:
-    bool mMXP;
-    TMXPMode mMXP_MODE;
-    TMXPMode mMXP_DEFAULT;
-
-    // MXP delegated handlers
-    TMxpNodeBuilder mMxpTagBuilder;
-    TMxpTagProcessor mMxpTagProcessor;
-    TEntityHandler mEntityHandler;
-
-    TMxpClient* mpMxpClient;
-
 public:
     explicit TMxpProcessor(TMxpClient* pMxpClient)
-    : mMXP(false)
-    , mMXP_MODE(MXP_MODE_OPEN), mMXP_DEFAULT(MXP_MODE_OPEN)
-    , mMxpTagBuilder(true), mEntityHandler(mMxpTagProcessor.getEntityResolver()), mpMxpClient(pMxpClient)
+    : mMxpTagBuilder(true) // Argument value means "ignoreText"
```

Comment:
not sure we need this comment

Reply:
I think it is helpful - it is forcing the construction of the instance of a separate class (`TMxpNodeBuilder`) to take a **non**-default action (the constructor is expressed as: `explicit TMxpNodeBuilder(bool ignoreText = false);` in *its* header) - and the MXP stuff is really opaque enough as it is...

Reply:
It takes five seconds to look up what the argument is named, we can't duplicate such things around!

Reply:
Well, it took me longer, but since you insist I'll strip it out... :slightly_frowning_face: